From 86d4ff2da5a9d2f8f056ffa3fbbfbcd8c8068f3e Mon Sep 17 00:00:00 2001
From: Sebastian Walderich <sebastian@walderich.de>
Date: Mon, 23 Jul 2012 09:35:35 +0200
Subject: [PATCH] Redirect ShowWithEffect to Show without DynamicLoader

Under Windows the method ShowWithEffect needs the class wxDynamicLoader. If
this class is disabled, the call of this method is now redirected to the
effectless method Show. So the user does not need to change the
implementation.
---
 src/msw/window.cpp |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/msw/window.cpp b/src/msw/window.cpp
index 441e258..34e2e2b 100644
--- a/src/msw/window.cpp
+++ b/src/msw/window.cpp
@@ -670,6 +670,7 @@ wxWindowMSW::MSWShowWithEffect(bool show,
                                wxShowEffect effect,
                                unsigned timeout)
 {
+#if wxUSE_DYNLIB_CLASS
     if ( effect == wxSHOW_EFFECT_NONE )
         return Show(show);
 
@@ -766,6 +767,9 @@ wxWindowMSW::MSWShowWithEffect(bool show,
     }
 
     return true;
+#else    // wxUSE_DYNLIB_CLASS
+    return Show(show);
+#endif
 }
 
 // Raise the window to the top of the Z order
-- 
1.7.10.msysgit.1

